addons/product/static/src/scss/report_label_sheet.scss

```diff
@@ -72,8 +72,8 @@
         padding:0;
         line-height:1;
         font-size:55%;
-        overflow:visible;
-        white-space:normal;
-        word-wrap: break-word;
```

Comment:
This change is also in v16, so the same bug should occur there as well. It would be cleaner to do a PR that reverts the commit that added this in (i.e. with the [REV] tag + git revert [commit] command with an explanation as to why you're reverting it) targeting v16 and then either its fw-port adds in an extra commit with the `report_lot_barcode.xml` fix or open up a second PR that fixes `report_lot_barcode.xml`